### PR TITLE
chore: 💄 Prettier all files

### PR DESCRIPTION
**Pull request by afc163**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

- English Changelog:
- Chinese Changelog (optional):

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered .github/ISSUE_TEMPLATE/----please-use-new-issue-ant-design---.md](https://github.com/ant-design/ant-design/blob/prettier-all/.github/ISSUE_TEMPLATE/----please-use-new-issue-ant-design---.md)
[View rendered .github/PULL_REQUEST_TEMPLATE.md](https://github.com/ant-design/ant-design/blob/prettier-all/.github/PULL_REQUEST_TEMPLATE.md)
[View rendered .github/PULL_REQUEST_TEMPLATE/pr_cn.md](https://github.com/ant-design/ant-design/blob/prettier-all/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)
[View rendered CODE_OF_CONDUCT.md](https://github.com/ant-design/ant-design/blob/prettier-all/CODE_OF_CONDUCT.md)
[View rendered README-zh_CN.md](https://github.com/ant-design/ant-design/blob/prettier-all/README-zh_CN.md)
[View rendered README.md](https://github.com/ant-design/ant-design/blob/prettier-all/README.md)
[View rendered components/affix/demo/basic.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/affix/demo/basic.md)
[View rendered components/affix/demo/on-change.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/affix/demo/on-change.md)
[View rendered components/affix/demo/target.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/affix/demo/target.md)
[View rendered components/affix/index.en-US.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/affix/index.en-US.md)
[View rendered components/affix/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/affix/index.zh-CN.md)
[View rendered components/alert/demo/banner.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/alert/demo/banner.md)
[View rendered components/alert/demo/basic.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/alert/demo/basic.md)
[View rendered components/alert/demo/closable.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/alert/demo/closable.md)
[View rendered components/alert/demo/close-text.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/alert/demo/close-text.md)
[View rendered components/alert/demo/custom-icon.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/alert/demo/custom-icon.md)
[View rendered components/alert/demo/description.md](https://github.com/ant-design/ant-design/blob/prettier-all/components/alert/demo/description.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -230,7 +230,7 @@
     "lint-staged:ts": "tsc && node node_modules/tslint/bin/tslint",
     "lint-staged:es": "eslint ./.*.js ./webpack.config.js",
     "lint-staged:demo": "cross-env RUN_ENV=DEMO eslint --ext '.md'",
-    "prettier": "node ./scripts/prettier.js"
+    "prettier": "prettier -c --write '**/*'"
```
Comment (afc163) [inline package.json]: `scripts/prettier.js` 不是必要的，自带的命令行够用了，也快很多。

File: package.json [hunk 2]
```diff
@@ -238,17 +238,17 @@
...
     "{tests,site,scripts,components}/**/*.{js,jsx}": [
       "npm run lint-staged:es",
-      "node ./scripts/prettier.js --pre",
+      "prettier --write",
       "git add"
     ],
     "{site,components}/**/*.less": [
       "stylelint --syntax less",
-      "node ./scripts/prettier.js --pre",
+      "prettier --write",
```
Comment (afc163) [inline package.json]: 提交前 prettier

Comment (chenshuai2144) [reply package.json]: 这样会 prettier 全部的

File: package.json (hunk 1 above)
Comment (chenshuai2144) [reply package.json]: 原来 md 不能 prettier 现在都可以了。

File: package.json (hunk 2 above)
Comment (afc163) [reply package.json]: 不会的，lint-staged 的原理是把当前修改的文件路径加到这些命令的背后，prettier 和 git add 都是这样。

Comment (afc163) [reply package.json]: https://github.com/okonet/lint-staged#reformatting-the-code

**Pull request merged**